### PR TITLE
Update docs/about.md to reflect the current team

### PR DESCRIPTION
**Pull request by damithc**
**What is the purpose of this pull request?**

- [x] Documentation update

**Overview of changes:**

Updated the list of team members to reflect the current status.

**Anything you'd like to highlight / discuss:**

Check if I missed anyone. Feel free to update further before merging.

**Testing instructions:**

**Proposed commit message: (wrap lines at 72 characters)**

```
Update docs/about.md to reflect the current team
```

---

### CODE REVIEW

File: docs/about.md [hunk 1]
```diff
@@ -6,39 +6,45 @@
...
+* [**Jia Zhixin**](https://github.com/nusjzx): _Member_ for May 2018 - Aug 2018
 * [**Jiang Sheng**](https://github.com/Gisonrg): _Founding Member_ and _Team Lead_ for Aug 2016 - Jul 2017
-* [**Lin Si Jie**](https://www.github.com/sijie123): _Project Member_ for Jan 2019 - Aug 2019
-* [**Lu Yang Kenneth**](https://github.com/luyangkenneth): _Project Member_ for Jan 2019 - Aug 2019
-* [**Maddi Aadyaa**](https://www.github.com/amad-person): _Project Member_ for Jan 2019 - Aug 2019
-* [**Rachael Sim Hwee Ling**](https://github.com/rachx): _Project Member_ for Jan 2018 - Dec 2018
-* [**Tan Zhen Yong**](https://www.github.com/Xenonym): _Project Member_ for Jan 2019 - Aug 2019
+* [**Joshua Wong**](https://www.github.com/hcwong): _Contributor_ for Jan-April 2020
+* [**Labayna Neil Brian Narido**](https://www.github.com/nbriannl): _Member_ for Jan - Apr 2020
+* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Mentor_ for Jan - Apr 2020, _Member_ for Jan 2019 - Aug 2019
```
Comment (ang-zeyu) [inline docs/about.md:36]: Think this should be 

```
* [**Marvin Chin Kun Song**](https://www.github.com/marvinchin): _Project Lead_ for Jan - Aug 2020, _Member_ for Jan 2019 - Aug 2019
```
🤔  

Wonder if we should switch to one of these for non-lead roles as well, might miss something out here
https://github.com/all-contributors/all-contributors

Comment (ang-zeyu) [reply docs/about.md:36]: > Wonder if we should switch to one of these for non-lead roles as well, might miss something out here

after this pr*

Comment (ang-zeyu) [reply docs/about.md:36]: *I'm not exactly too sure how these work or if they are even automatable though 😅 (I'm assuming they are)

Comment (damithc) [reply docs/about.md:36]: We can certainly give it a try. 👍

Comment (ang-zeyu) [reply docs/about.md:36]: 👍 

up https://github.com/MarkBind/markbind/issues/1646

**Comment by damithc** [comment]
> Noting that the all-contributors bot is now used, can I suggest closing this PR?

This page remains woefully out-of-date though https://markbind.org/about.html

**Comment by kaixin-hc** [comment]
@damithc That's a good point, but this PR isn't up to date either - perhaps it would be better suited as an issue then? Or you or someone else can update it and we can merge it in.

**Comment by damithc** [comment]
Sure, I'm fine as long as the page gets updated.

**Pull request closed without merging**